### ISSUE TITLE
maximal_polyhedra of PolyhedralComplexes produces warnings and broken polyhedra. 

### ISSUE THREAD

**Issue by YueRen** [oscar-system/Oscar.jl#1715]
Calling `maximal_polyhedra` results in warnings and broken polyhedra, calling `vertices` before `maximal_polyhedra` fixes it. The code
```
using Oscar
R,(x1,x2,x3,x4,x5,x6) = PolynomialRing(ZZ,6)
I = ideal(R,[-x1+x3+x4,-x2+x3+x5,-x1+x2+x6])
val = TropicalSemiringMap(ZZ)
X = TropicalLinearSpace(I,val)
Sigma = X.polyhedralComplex
# vertices(Sigma) # running this line fixes the warnings and errors below
S = maximal_polyhedra(Sigma)
```
produces following warnings:
```
15-element SubObjectIterator{Polyhedron{fmpq}}:
polymake:  WARNING: available properties insufficient to compute 'FACETS'
polymake:  WARNING: available properties insufficient to compute 'LINEAR_SPAN'
[...]
polymake:  WARNING: available properties insufficient to compute 'FACETS'
polymake:  WARNING: available properties insufficient to compute 'LINEAR_SPAN'
A polyhedron in ambient dimension 6
 polymake:  WARNING: available properties insufficient to compute 'FACETS'
polymake:  WARNING: available properties insufficient to compute 'LINEAR_SPAN'
polymake:  WARNING: available properties insufficient to compute 'FACETS'
polymake:  WARNING: available properties insufficient to compute 'LINEAR_SPAN'
A polyhedron in ambient dimension 6
 polymake:  WARNING: available properties insufficient to compute 'FACETS'
polymake:  WARNING: available properties insufficient to compute 'LINEAR_SPAN'
polymake:  WARNING: available properties insufficient to compute 'FACETS'
polymake:  WARNING: available properties insufficient to compute 'LINEAR_SPAN'
A polyhedron in ambient dimension 6
 polymake:  WARNING: available properties insufficient to compute 'FACETS'
polymake:  WARNING: available properties insufficient to compute 'LINEAR_SPAN'
polymake:  WARNING: available properties insufficient to compute 'FACETS'
polymake:  WARNING: available properties insufficient to compute 'LINEAR_SPAN'
A polyhedron in ambient dimension 6
 polymake:  WARNING: available properties insufficient to compute 'FACETS'
polymake:  WARNING: available properties insufficient to compute 'LINEAR_SPAN'
polymake:  WARNING: available properties insufficient to compute 'FACETS'
polymake:  WARNING: available properties insufficient to compute 'LINEAR_SPAN'
A polyhedron in ambient dimension 6
 polymake:  WARNING: available properties insufficient to compute 'FACETS'
polymake:  WARNING: available properties insufficient to compute 'LINEAR_SPAN'
polymake:  WARNING: available properties insufficient to compute 'FACETS'
polymake:  WARNING: available properties insufficient to compute 'LINEAR_SPAN'
A polyhedron in ambient dimension 6
 polymake:  WARNING: available properties insufficient to compute 'FACETS'
polymake:  WARNING: available properties insufficient to compute 'LINEAR_SPAN'
polymake:  WARNING: available properties insufficient to compute 'FACETS'
polymake:  WARNING: available properties insufficient to compute 'LINEAR_SPAN'
A polyhedron in ambient dimension 6
 polymake:  WARNING: available properties insufficient to compute 'FACETS'
polymake:  WARNING: available properties insufficient to compute 'LINEAR_SPAN'
polymake:  WARNING: available properties insufficient to compute 'FACETS'
polymake:  WARNING: available properties insufficient to compute 'LINEAR_SPAN'
A polyhedron in ambient dimension 6
 polymake:  WARNING: available properties insufficient to compute 'FACETS'
polymake:  WARNING: available properties insufficient to compute 'LINEAR_SPAN'
polymake:  WARNING: available properties insufficient to compute 'FACETS'
polymake:  WARNING: available properties insufficient to compute 'LINEAR_SPAN'
A polyhedron in ambient dimension 6
 polymake:  WARNING: available properties insufficient to compute 'FACETS'
polymake:  WARNING: available properties insufficient to compute 'LINEAR_SPAN'
polymake:  WARNING: available properties insufficient to compute 'FACETS'
polymake:  WARNING: available properties insufficient to compute 'LINEAR_SPAN'
A polyhedron in ambient dimension 6
 polymake:  WARNING: available properties insufficient to compute 'FACETS'
polymake:  WARNING: available properties insufficient to compute 'LINEAR_SPAN'
polymake:  WARNING: available properties insufficient to compute 'FACETS'
polymake:  WARNING: available properties insufficient to compute 'LINEAR_SPAN'
A polyhedron in ambient dimension 6
 polymake:  WARNING: available properties insufficient to compute 'FACETS'
polymake:  WARNING: available properties insufficient to compute 'LINEAR_SPAN'
polymake:  WARNING: available properties insufficient to compute 'FACETS'
polymake:  WARNING: available properties insufficient to compute 'LINEAR_SPAN'
A polyhedron in ambient dimension 6
 polymake:  WARNING: available properties insufficient to compute 'FACETS'
polymake:  WARNING: available properties insufficient to compute 'LINEAR_SPAN'
polymake:  WARNING: available properties insufficient to compute 'FACETS'
polymake:  WARNING: available properties insufficient to compute 'LINEAR_SPAN'
A polyhedron in ambient dimension 6
 polymake:  WARNING: available properties insufficient to compute 'FACETS'
polymake:  WARNING: available properties insufficient to compute 'LINEAR_SPAN'
polymake:  WARNING: available properties insufficient to compute 'FACETS'
polymake:  WARNING: available properties insufficient to compute 'LINEAR_SPAN'
A polyhedron in ambient dimension 6
 polymake:  WARNING: available properties insufficient to compute 'FACETS'
polymake:  WARNING: available properties insufficient to compute 'LINEAR_SPAN'
polymake:  WARNING: available properties insufficient to compute 'FACETS'
polymake:  WARNING: available properties insufficient to compute 'LINEAR_SPAN'
A polyhedron in ambient dimension 6
```
and also following errors when I try to use the polyhedra:
```
julia> dim(S[1])
polymake:  WARNING: available properties insufficient to compute 'FACETS'
polymake:  WARNING: available properties insufficient to compute 'LINEAR_SPAN'
polymake:  WARNING: available properties insufficient to compute 'COMBINATORIAL_DIM'
```
But running the commented line with `vertices(Sigma)` fixes it.

**Comment by lkastner** [comment]
The bug comes from polymake, as `Sigma` is really a `TropicalCycle` (I think) and at construction only has the property `PROJECTIVE_VERTICES`. Every `TropicalCycle` is also a `PolyhedralComplex` and there are rules to compute `VERTICES`, etc. from the `PROJECTIVE_VERTICES`. However, the method returning the n-th maximal cone/polyhedron tries to be clever and wants to avoid triggering of rules. Hence it looks up which properties are present and only takes those that are present to construct this polyhedron. However it works only on the `PolyhedralComplex` part, so it does not take the `PROJECTIVE_VERTICES` into consideration, hence you get back almost empty objects. Your manual triggering of `vertices` obviously solves the issue, I hope that is fine for the time being.

We are debating whether we want to attach a different method for computing the maximal cells of a `TropicalCycle` or how exactly to solve this issue. However it will need to be solved in polymake, so it may take some time until it arrives in Oscar.

**Comment by YueRen** [comment]
Manually triggering `vertices` works for me, glad to hear that this reliably fixes the issue!

**Comment by micjoswig** [comment]
@YueRen With the new OSCAR 0.11.3 I cannot reproduce the error any more.  Can you please check and report back?  Close the issue if problem is solved.

**Comment by YueRen** [comment]
I can confirm that the error has been fixed with OSCAR 0.11.3,
